### DISCUSSION TITLE
How can I apply Unity's Baked Lightmap in Filament?

### DISCUSSION THREAD

**Discussion by CYH0613** [google/filament#7204] [General]
I have a Baked Lightmap from Unity that I want to use in Filament. I'm currently trying to apply the lightmap in my custom Filament shader, but the result isn't matching what I see in Unity.

I've got the tiling and offset values from Unity and applied them to the UV coordinates in my shader. However, the result still doesn't look right. Is there any specific process I should follow to correctly map Unity's Baked Lightmap to my models in Filament? Any help would be appreciated.

**Comment by CYH0613** [comment]
@romainguy

**Reply by romainguy** [reply]
I am not familiar with Unity's lightmap. There are many ways to encode lightmaps. Are the lightmaps you are referring to just diffuse maps?
